.github/workflows/main.yml

```
@@ -26,6 +26,7 @@ env:
 
 jobs:
   ubuntu:
+    if: "!endsWith(github.event.pull_request.title, '[ci-windows]')"
```

I think it would be better not to add a token referring to one platform (`windows` here) that affects CI for other system(s) (`linux` / `ubuntu`).

It's fine to begin with, but it gets complicated if the combination of possible platforms expands in future.  Also, people could misunderstand the purpose of it (for example, when I expected the `windows-testing` branch to enable additional tests because I'd forgotten the design for it, to disable non-Windows tests).

What do you think of:

```suggestion
    if: "!endsWith(github.event.pull_request.title, '[ci-skip-linux]')"
```